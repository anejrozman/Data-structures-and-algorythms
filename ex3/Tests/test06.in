15,56,5
1.669615289456551,4.480174137636233,3.9922648696358376,3.5058601231894384,6.372229222836898,2.440835635479904,6.553093502956248,7.157523299150654,7.679015890007595,2.349524882176882,6.721155683653381,7.668607021875721,6.661883368363209,0.5122077869356929,2.280173215263284
10,2,640.614
9,3,643.707
1,7,899.297
5,14,901.166
2,9,263.2
0,8,391.383
13,9,777.2
1,8,397.619
3,12,398.809
4,14,15.359
1,10,276.742
2,6,660.513
0,3,278.716
3,11,794.29
14,0,155.331
7,5,284.992
0,1,541.909
4,13,799.433
12,5,544.612
9,10,673.504
8,10,806.891
5,13,422.774
0,13,683.675
3,1,302.108
6,3,305.745
10,14,178.038
11,1,817.29
2,3,695.244
8,7,698.077
14,9,699.022
8,4,317.749
2,0,574.842
1,6,191.345
10,0,708.57
10,3,455.689
9,4,456.855
14,8,591.321
3,13,850.796
5,4,852.825
10,13,466.295
13,14,728.119
12,4,347.073
14,2,94.494
5,3,735.093
12,0,869.601
7,4,874.878
12,14,619.017
7,13,749.747
9,0,111.461
7,11,365.249
6,12,237.382
7,2,754.885
11,0,243.542
8,13,884.903
11,2,245.245
11,9,895.865
