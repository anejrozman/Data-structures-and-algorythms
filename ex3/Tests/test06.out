11,1218.4113
1,0
1,0
1,0
1,0
